build-tools/packages/build-cli/src/test/data/exports/exports.d.ts

```diff
@@ -10,11 +10,13 @@ export declare type A = number;
 /**
  * @public
  */
+// eslint-disable-next-line one-var
```

Comment:
I assume this is the reason? Same for lines 21-22?
```suggestion
// eslint-disable-next-line one-var -- By design for testing this case
```